plugins/kubernetes/app/views/kubernetes_roles/_roles_list.html.erb

```diff
@@ -0,0 +1,41 @@
+<script type="text/ng-template" id="kubernetes/kubernetes_roles.tmpl.html">
+  <div>
+    <table class="table table-condensed">
+      <thead>
+      <tr>
+        <th>Role</th>
+        <th>Config File</th>
+        <th>Service Name</th>
+        <th>RAM</th>
+        <th>CPU</th>
+        <th>Replicas</th>
+        <th>Actions</th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <tr ng-show="!roles.length" ng-hide="roles.length">
+        <td>No project roles have been created yet.</td>
```

Comment:
Does this not look weird to have this text squashed into the first column?
If it does look weird, maybe extract this outside the table and only show the table if there are roles.